publishedbranches/docs-mongocli.yaml

```diff
@@ -27,13 +29,14 @@ version:
     - '1.2'
     - '1.1'
     - '1.0'
-  stable: '1.8'
-  upcoming: '1.9'
+  stable: 'v1.9'
+  upcoming: 'v1.10'
```

Comment:
Should I just remove this line if snooty doesn't use it?

Reply:
You could keep upcoming for now as Jon is considering if we want to display it in the future

Reply:
Sounds good, thanks.